recipe/meta.yaml

```diff
@@ -25,12 +25,6 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - bazel =4.2.2
-    - cython >=0.29.26
-    - curl
-    - make
-    - m2-patch       # [win]
-    - python
```

Comment:
@h-vetinari could you please explain why these are no longer needed?

Reply:
Because everything is handled per output & the build succeeds without